src/main/java/edu/oregonstate/cs361/battleship/Ship.java

```diff
@@ -11,6 +11,16 @@
     private Coordinate start;
     private Coordinate end;
 
+    public Ship() {
+        name = "";
```

Comment:
Another way to specify a default constructor when you already have a custom one is to do something like: 
```
public Ship() {}
```
http://stackoverflow.com/questions/4488716/java-default-constructor